TCAT/AppDelegate.swift

```diff
@@ -28,6 +28,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CLLocationManagerDelegate
         
         // Initalize window without storyboard
         let rootVC = HomeViewController()
+        rootVC.getBusStops()
```

Comment:
Ideally it would be here. I feel like we are a point where we can still test our features without changing the rootVC.